src/renderer/angular/directives/commons/sk-circle-pie-chart.directive.js

```diff
@@ -39,14 +39,14 @@ function SkCirclePieChartDirective($timeout, CommonService) {
 
                 scope.totalValueFormated = new BigNumber(0);
                 items.forEach((item) => {
-                    item.valueUSD = Number(CommonService.numbersAfterComma(item.valueUSD, 2));
-                    scope.totalValueFormated = scope.totalValueFormated.plus(new BigNumber(item.valueUSD));
+                    item.unformattedValueUSD = CommonService.formattedLocaleNumberToNumber(item.valueUSD);
```

Comment:
If we have to do this we're formatting too early.

There should only be a single conversion, from unformatted to formatted, as late as possible, just before the value is rendered to the screen.